apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -488,40 +486,44 @@ export default {
     refLinkCandidates() {
       return this.table.columns
         .filter(
-          (c) =>
+          (c: Record<string, any>) =>
             (c.columnType === "REF" || c.columnType === "REF_ARRAY") &&
-            c.name !== this.modelValue.name
+            c.name !== this.modelValue?.name
         )
-        .map((c) => c.name);
+        .map((c: Record<string, any>) => c.name);
     },
-    refBackCandidates(fromTable, toTable) {
+    refBackCandidates(
+      fromTable: Record<string, any>,
+      toTable: Record<string, any>
+    ) {
       const schema =
         this.refSchema !== undefined ? this.refSchema : this.schema;
 
       const columns = schema.tables
-        .filter((t) => t.name === fromTable)
-        .map((t) => t.columns)[0];
+        .filter((t: Record<string, any>) => t.name === fromTable)
```

Comment:
It's not, because the types here are still very vague (Record<string, any>), since the object are being build here we can't use the meta-data typings yet. That would require a bit more refactoring and thinking, I think.